web/__init__.py

```diff
@@ -1,61 +1,83 @@
 import json
+import tempfile
 import logging as log
 
-from flask import Flask, request, render_template, Response
+from secrets import token_urlsafe as token
+from flask import Flask, request, render_template, Response, session, url_for
 from flask_sock import Sock
-
-from libflagship.pppp import P2PSubCmdType, FileTransfer
-from libflagship.ppppapi import FileUploadInfo, PPPPError
-
+from user_agents import parse as user_agent_parse
 
 from web.lib.service import ServiceManager
 
+import web.config
+import web.platform
+import web.util
+
 import cli.util
+import cli.config
+
 
 app = Flask(
     __name__,
     root_path=".",
     static_folder="static",
     template_folder="static"
 )
+# secret_key is required for flash() to function
+app.secret_key = token(24)
 app.config.from_prefixed_env()
 app.svc = ServiceManager()
 
 sock = Sock(app)
 
-
-import web.service.pppp
-import web.service.video
-import web.service.mqtt
-import web.service.filetransfer
+import web.service.pppp  # nopep8
+import web.service.video  # nopep8
+import web.service.mqtt  # nopep8
+import web.service.filetransfer  # nopep8
 
 
 @app.before_first_request
 def startup():
     app.svc.register("pppp", web.service.pppp.PPPPService())
     app.svc.register("videoqueue", web.service.video.VideoQueue())
     app.svc.register("mqttqueue", web.service.mqtt.MqttQueue())
-    app.svc.register("filetransfer", web.service.filetransfer.FileTransferService())
+    app.svc.register(
```

Comment:
Is this auto-reformatting? Please undo this line's change.

Reply:
Can do, and yes it was, due to line length, but I agree this makes it ugly :D

Reply:
✅